test/types/assoc.ts

```diff
@@ -0,0 +1,27 @@
+import { expectType } from "tsd";
+import { __, assoc } from "../../es/index";
+import { AssocPartialOne} from '../../types/util/tools';
+
+type Point = { x: number; y: number };
+const p: Point = { x: 2, y: 3 };
+
+const t = assoc('x')(5, p);
```

Comment:
Does this line do something?

Reply:
No, I was using it to test, forgot to remove